README.md

```diff
@@ -2,32 +2,52 @@
 
 SPPNaut est une startup d'Etat dont la mission est la modernisaton et l'ouverture des publications nautiques.
 
-
 ## Dévéloppement
 
 ### Prerequis
 
 Pour faire tourner l'interface en local, il est conseiller d'utiliser :
-* un environnement virtuel pour faire tourner python >= 3.10
-* pipenv (voir pour passer à piplock ?)
-* docker et docker-compose pour faire tourner la base de données Postgresql
+
+-   Python >= 3.10
+-   docker et docker-compose pour faire tourner la base de données Postgresql
 
 ### Installation
 
-Installation des dépendances :
+1. Création d'un environnement virtuel
+
+    ```sh
+    python -m venv .venv --prompt $(basename $(pwd))
```

Comment:
Ça c'est la méthode Anthony ! ha ha ha 

J'aurais juste noté ici qu'on a besoin d'un environnement virtuel en 3,10,4

Reply:
:+1: